src/components/DiagramFrame/DiagramFrame.vue

```diff
@@ -39,12 +39,8 @@
       </div>
       <div class="footer text-skin-control px-4 py-1 flex justify-between">
         <div class="flex items-center gap-3 color-base">
-          <button class="bottom-1 left-1 hide-export" @click="showTipsDialog()">
-            <svg class="filter grayscale w-4 h-4" viewBox="0 0 1024 1024" xmlns="http://www.w3.org/2000/svg">
-              <path
-                d="M512 1024C229.248 1024 0 794.752 0 512S229.248 0 512 0s512 229.248 512 512-229.248 512-512 512z m0-938.666667C276.352 85.333333 85.333333 276.352 85.333333 512s191.018667 426.666667 426.666667 426.666667 426.666667-191.018667 426.666667-426.666667A426.666667 426.666667 0 0 0 512 85.333333z m0 682.666667a42.666667 42.666667 0 0 1-42.368-42.666667v-255.573333a42.368 42.368 0 1 1 84.693333 0V725.333333A42.410667 42.410667 0 0 1 512 768z m0-426.666667a42.325333 42.325333 0 1 1-0.085333-84.650666A42.325333 42.325333 0 0 1 512 341.333333z m42.325333-42.666666v0z"
-                fill="currentColor"></path>
-            </svg>
+          <button class="bottom-1 flex items-center left-1 hide-export" @click="showTipsDialog()">
+            <Icon icon-class="filter grayscale w-4 h-4" name="tip"/>
```

Comment:
Probably move `name` to the first position of attributes to improve readability. It is not a blocker for this PR.